pkg/tsdb/grafanads/grafana.go

```diff
@@ -165,7 +165,11 @@ func (s *Service) doReadQuery(ctx context.Context, query backend.DataQuery) back
 func (s *Service) doRandomWalk(query backend.DataQuery) backend.DataResponse {
 	response := backend.DataResponse{}
 
-	model := testdatasource.JSONModel{}
+	model, err := testdatasource.GetJSONModel(nil)
```

Comment:
Nit: Would prefer here:
```suggestion
	model, err := testdatasource.GetJSONModel(json.RawMessage{})
```

Alternatively, keep as is and add a comment to `GetJSONModel` just in case someone in the future doesn't catch this